examples/sw/led/led.c

```diff
@@ -37,7 +37,7 @@ static int usleep(unsigned long usec) {
 int main(int argc, char **argv) {
   // Any data written to the stack segment will connect the lowest four bits to
   // the board leds
```

Comment:
Does that comment need updating?

Reply:
It does, thanks for catching this Philipp.